client/src/features/ProjectPageV2/utils/roleUtils.ts

```diff
@@ -51,9 +52,22 @@ export function toSortedMembers(members: ProjectMemberListResponse) {
     if (a.role !== b.role) {
       return toNumericRole(b.role) - toNumericRole(a.role);
     }
-    if (a.email && b.email) {
-      return a.email.localeCompare(b.email);
+    if (a.first_name && b.first_name) {
+      return a.first_name.localeCompare(b.first_name);
     }
     return a.id < b.id ? -1 : 1;
   });
 }
+
```

Comment:
3/4 users of `getMemberNameToDisplay` display it in the same way, so I suggest making this a `.tsx` file and including a component:

```suggestion
interface ProjectMemberDisplayProps {
  member: ProjectMemberResponse;
  nameInBold?: boolean;
}

export function ProjectMemberDisplay({
  member,
  nameInBold = false,
}: ProjectMemberDisplayProps) {
  const displayMemberName = getMemberNameToDisplay(member);
  const memberName = nameInBold ? (
    <b>{displayMemberName}</b>
  ) : (
    displayMemberName
  );

  return (
    <>
      {memberName}{" "}
      {member.namespace && (
        <span className="fst-italic">@{member.namespace}</span>
      )}
    </>
  );
}

```